File: Submissions/#0446_div2/A/main.java
import java.util.*;

public class Main {

  public static void main (String[] args) {

    Scanner sc = new Scanner(System.in);

    int n = sc.nextInt();

    long sum = 0;
    for (int i = 0; i < n; i++) {
      sum += sc.nextInt();
    }

    int [] b = new int[n];
    for (int i = 0; i < n; i++) {
      b[i] = sc.nextInt();
    }
  
    sc.close();

    Arrays.sort(b);

    String answer = "YES";

    if (n != 2) {
      if ((b[n - 1] + b[n - 2]) < sum) {
        answer = "NO";
      }
    }

    System.out.println(answer);

  }

}

File: Submissions/#0446_div2/A/main.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {

  int n;
  cin >> n;

  long long sum = 0;
  for (int i = 0; i < n; i++) {
    int a;
    cin >> a;
    sum += a;
  }

  vector<long long> b(n);
  for (int i = 0; i < n; i++) {
    cin >> b[i];
  }
  sort(b.begin(), b.end());

  string answer = "YES";

  if (n != 2) {
    if ((b[n - 1] + b[n - 2]) < sum) {
      answer = "NO";
    }
  }
  
  cout << answer << endl;

}